scripts/macos-instructions.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+
+set -o errexit
+set -o pipefail
+set -o nounset
+# set -o xtrace
+
+# This script requires yq (https://github.com/mikefarah/yq) which can be installed with this command
+# snap install yq
```

Comment:
Snap isn't on a Mac, you could have brew instructions too.

Reply:
I have added some brew instructions.

[c5b6be1](https://github.com/dockstore/dockstore/pull/5043/commits/c5b6be1ca49572d311e4b4bcd4c935c87c180714)